ddl/table.go

```diff
@@ -96,7 +97,13 @@ func onCreateView(d *ddlCtx, t *meta.Meta, job *model.Job) (ver int64, _ error)
 		// none -> public
 		tbInfo.State = model.StatePublic
 		tbInfo.UpdateTS = t.StartTS
-		err = t.CreateTable(schemaID, tbInfo)
+		if oldTbInfoID > 0 && orReplace {
+			err = t.DropTable(schemaID, oldTbInfoID, true)
```

Comment:
should we also change the function name from "DropTable" to "DropTableOrView"?

Reply:
Yes, `DropView` has been merge now.

Reply:
@zz-jason Done